.github/workflows/ci.yml

```diff
@@ -36,6 +38,35 @@ jobs:
       - run: cargo build --features serde
       - run: cargo test --features serde
 
+  test-features-default-no-grid-nor-flexbox:
```

Comment:
The key of this CI job probably ought to be renamed as it does not use default features.